packages/tab/src/index.d.ts

```diff
@@ -0,0 +1 @@
+export { default } from './react/index.js'
```

Comment:
This is necessary to make TypeScript aware of the re-export and so it can pick up the types.